src/main/java/com/tutorhub/service/impl/ContentServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.tutorhub.service.impl;
+
+import com.tutorhub.exception.InvalidFormatException;
+import com.tutorhub.exception.ResourceAlreadyExistsException;
+import com.tutorhub.model.course.ContentType;
+import com.tutorhub.s3.client.service.AwsS3Service;
+import com.tutorhub.service.ContentService;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+@RequiredArgsConstructor
+public class ContentServiceImpl implements ContentService {
+  private final AwsS3Service awsS3Service;
+
+  @Override
+  public Object find(String filename) {
```

Comment:
what would be if not found? exception of null?

Reply:
Oops, throws an exception

Reply:
Fixed